hail/src/main/scala/is/hail/expr/types/physical/PArrayBackedContainer.scala

```diff
@@ -110,6 +120,9 @@ trait PArrayBackedContainer extends PContainer {
   def loadElement(aoff: Long, length: Int, i: Int): Long =
     arrayRep.loadElement(aoff, length, i)
 
+  def loadElementAddress(aoff: Code[Long], length: Code[Int], i: Code[Int]): Code[Long] =
```

Comment:
needed to be called something other than loadElement, because scala cannot distinguish between Code[Region] and Code[Long] types in the argument list, and besides this (arity, other argument types), this method is identical to loadElement(region: Code[Region] ...), leading to type check errors.

Later on, in the PR corresponding to #7826, the region-bearing methods will go away.
references https://github.com/hail-is/hail/issues/7829